### ISSUE TITLE
Use of deprecated sbt-assembly key breaks build of freshly created project

### ISSUE THREAD

**Issue by srfc** [spotify/scio.g8#167]
# Summary

As far as I can tell `sbt-assembly` deprecated the `jarName` key with release `1.0.0`.
This template almost always uses its replacement `assemblyJarName` except in [`build.sbt`'s L125](https://github.com/spotify/scio.g8/blob/c0d7d9c599e3c1ad550102f4abd3a16e5e9a009e/src/main/g8/build.sbt#L125).

When creating a new project using 
```
sbt new spotify/scio.g8
// redacted
name [scio job]: example
organization [example]: com.example
DataflowRunner [yes/NO]: yes
DataflowFlexTemplate [yes/NO]: yes
FlinkRunner [yes/NO]: no
SparkRunner [yes/NO]: no
```
it prevents `sbt` from successfully building the fresh project.

# Suggestion

Change [`build.sbt`'s L125](https://github.com/spotify/scio.g8/blob/c0d7d9c599e3c1ad550102f4abd3a16e5e9a009e/src/main/g8/build.sbt#L125) to
```
scriptClasspath := Seq((assembly / assemblyJarName).value),
```
to reflect `sbt-assembly`'s API changes.

# Contribution

I couldn't find contributors guidelines on first try.

Will you accept a PR for this?

**Comment by regadas** [comment]
Hi @srfc Yeah please PR's are welcomed!

**Comment by kellen** [comment]
Closed by #168
